waku/v2/utils/flatten.nim

```diff
@@ -0,0 +1,12 @@
+when (NimMajor, NimMinor) < (1, 4):
```

Comment:
Perhaps `sequence.nim` for this module so that we can add other seq-related utilities here?

Reply:
sure! renamed and moved to /waku/common
53795549e81905251bd8acaf25acdb46a8001c5c